migrations/202106301200_added_allowing_email_auth.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE tenants ADD COLUMN is_allowing_email_auth BOOLEAN;
```

Comment:
I'd suggest `is_email_auth_allowed`, it's easier to follow that pattern in future if we need: `is_{something}_allowed` or `is_{something}_enabled` etc

Reply:
I agree, that name wasn't really pretty, it'll be better like this :smile: 
Renamed database column to `is_email_auth_allowed` in table `tenant`
Renamed field to `IsEmailAuthAllowed` in Go structures
Renamed field to `isEmailAuthAllowed` in JSON models
Renamed several functions using a related name in both back-end and front-end